src/domain/permissions.js

```diff
@@ -37,6 +39,9 @@ const rolesForPermission = {
   [RESTORE_DOC_REVISIONS]: [ROLE.admin],
   [MANAGE_ARCHIVED_DOCS]: [ROLE.admin],
   [MANAGE_IMPORT]: [ROLE.admin],
+  [MANAGE_EXPORT]: [], // This permission is restricted to a built-in user exclusively
```

Comment:
Should we rather reflect the comment in the name of the permission? I for one prefer that over comments, when possible.
Maybe `MANAGE_EXPORT_WITH_BUILT_IN_USER` ?

Reply:
Good idea, changed to `MANAGE_EXPORT_WITH_BUILT_IN_USER`.